lib/active_job/cancel/queue_adapters/test_adapter/rails.rb

```diff
@@ -1,13 +1,20 @@
 module ActiveJob
   module QueueAdapters
     class TestAdapter
+
       def job_to_hash(job, extras = {})
-        {
+        # These values should match up with what other ActiveJob::QueueAdapters publish. At minimum, job_class, job_id, arguments...
```

Comment:
How about reusing the original `job_to_hash`?  
I think that this can support to keep working when is added required parameters to `job_to_hash`.

```suggestion
    class TestAdapter
      alias original_job_to_hash job_to_hash

      def job_to_hash(job, extras = {})
        original_job_to_hash(job, extras).merge!({
          id: job.job_id,
          job: job.class,
          args: job.serialize.fetch('arguments'),
          queue: job.queue_name
        })
      end
```